### DISCUSSION TITLE
Is it possible to assign exceptions with pages' postfix? (.pdf)

### DISCUSSION THREAD

**Discussion by Cookie755233** [darkreader/darkreader#8172] [Q&A]
Hello, can I set pages that ends with .pdf (when it opens in browser) never use dark mode? Is that possible? Appreciated.

**Comment by Gusted** [comment]
Hi for chromium there's a button to enable/disable PDF's within dark reader, but requires the new design:

    Click Dark Reader icon.
    Click Dev tools (in the bottom-right corner).
    Click Preview new design.


Otherwise for other postfix you can use `.pdf$` within the sitelist.

**Reply by Cookie755233** [reply]
Brilliant! Have a wonderful day!
